tests/test.py

```diff
@@ -58,7 +70,10 @@ def test_proposals_with_unknown_node_key(self):
         self.assertEqual([], data['proposals'])
 
     def test_session_stats_create_without_session(self):
```

Comment:
test_session_stats_create_without_session() is misleading, because session key is provided.
I would suggest to rename function to test_session_stats_create_successful()

Reply:
Renamed